src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -32,6 +32,9 @@ export class SkyFileDropComponent {
   @Output()
   public filesChanged = new EventEmitter<SkyFileDropChange>();
 
+  @Output()
+  public linkInputBlurred = new EventEmitter<void>();
```

Comment:
Should we name this event in the present tense, `linkInputBlur` to indicate it can happen multiple times? We only would use the past-tense if the event happens once.

Reply:
Ummmm. Idk :) We use the past tense on all of our events on this component even when they are emitted multiple times.

But based on our style guide you are correct. I'll change it :)

Reply:
Done